googlemock/cmake/Config.cmake.in

```diff
@@ -0,0 +1,4 @@
+@PACKAGE_INIT@
+
+include("${CMAKE_CURRENT_LIST_DIR}/@targets_export_name@.cmake")
+check_required_components("@PROJECT_NAME@")
```

Comment:
See `no newline at end of file` warning. What editor you're using? I have a project with instructions to how to tune the environment to avoid such issues: http://0.readthedocs.org/en/latest/git.html#check-there-is-a-new-line-at-end-of-file, want to add more there.